spec/call_controllers/record_spec.rb

```diff
@@ -0,0 +1,87 @@
+    # If the caller is silent for more than five seconds,
+    # hits the # key, or the recording maxlength time is hit,
+    # Twilio will make an HTTP POST request to the default action URL
+    # (ie. the current document URL) with the parameters
+    # RecordingUrl and RecordingDuration.
+
+    it "records audio" do
+      controller = build_controller(stub_voice_commands: :record)
+      stub_twiml_request(controller, response: <<~TWIML)
+        <?xml version="1.0" encoding="UTF-8" ?>
+        <Response><Record/></Response>
+      TWIML
+
+      controller.run
+
+      expect(controller).to have_received(:record).with(async: true, start_beep: true)
+    end
+
+    describe "Verb Attributes" do
+      # From: https://www.twilio.com/docs/api/twiml/record
+
+      # The <Record> verb supports the following attributes that modify its behavior:
+
+      # | Attribute Name                | Allowed Values                 | Default Value        |
+      # |-------------------------------|--------------------------------|----------------------|
+      # | action                        | Relative or absolute URL       | current document URL |
+      # | method                        | GET, POST                      | POST                 |
+      # | timeout                       | Positive integer               | 5                    |
+      # | finishOnKey                   | Any digit, #, *                | 1234567890*#         |
+      # | maxLength                     | Integer greater than 1         | 3600 (1 hour)        |
+      # | playBeep                      | true, false                    | true                 |
+      # | trim                          | trim-silence, do-not-trim      | trim-silence         |
+      # | recordingStatusCallback       | relative or absolute URL       | None                 |
+      # | recordingStatusCallbackMethod | GET, POST                      | POST                 |
+      # | recordingStatusCallbackEvent  | in-progress, completed, absent | completed            |
+      # | transcribe                    | true, false                    | false                |
+      # | transcribeCallback            | Relative or absolute URL       | None                 |
+
+      describe "playBeep" do
+        # From: https://www.twilio.com/docs/api/twiml/record
+
+        # The `playBeep`` attribute allows you to toggle between playing a sound before the start of a recording.
+        # If you set the value to false, no beep sound will be played.
+
+        it "plays a beep" do
+          controller = build_controller(stub_voice_commands: :record)
+
+          stub_twiml_request(controller, response: <<~TWIML)
+            <?xml version="1.0" encoding="UTF-8" ?>
+            <Response><Record playBeep="true"/></Response>
+          TWIML
+
+          controller.run
+
+          expect(controller).to have_received(:record).with(hash_including(start_beep: true))
+        end
```

Comment:
Maybe because the default value is true anyway, we could delete this test because we already have assertion at the top.